packages/typespec-azure-resource-manager/lib/decorators.tsp

```diff
@@ -125,12 +125,14 @@ extern dec armResourceCollectionAction(target: Operation);
  * @param resourceType Resource
  * @param parentTypeName: Parent type name.
  * @param parentFriendlyTypeName Friendly name for parent.
+ * @param updateOperationName Whether to update the operation name in addition to the doc. Default: true
  */
 extern dec armRenameListByOperation(
   target: Operation,
   resourceType: Model,
   parentTypeName?: valueof string,
-  parentFriendlyTypeName?: valueof string
+  parentFriendlyTypeName?: valueof string,
+  updateOperationName?: valueof boolean
```

Comment:
this feels a little weird to have that flag when the decorator point is doing that

Reply:
also should we not have just removed it?

Reply:
True. Since it has two parts, doc and operation name and now I am reusing it to only do doc for part of the call.

Reply:
Alternative is do another private decorator and invoke the original internally with the optional flag.

Reply:
yeah my vote would be 
1. remove this decorator
2. make a new internal one that does the doc part